en/docs/extend/errors/error-codes-and-descriptions.md

```diff
@@ -0,0 +1,79 @@
+# Error Codes and Descriptions
+
+The following topic provides a list of error codes in the WSO2 Identity
```

Comment:
Don't you think it is better to not have fullstops for the descriptions that are phrases?